metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/resolver.py

```diff
@@ -772,12 +794,15 @@ def create_dataplatform_tables(
             qualified_table_name=qualified_table_name,
         )
 
-        return [
-            DataPlatformTable(
-                data_platform_pair=self.get_platform_pair(),
-                urn=urn,
-            )
-        ]
+        return Lineage(
+            upstreams=[
+                DataPlatformTable(
```

Comment:
I think the `data_platform_pair` probably should've been a mapping type or something - nothing to change for now, but we'll probably want to clean this up in the future

Reply:
ok